### PR TITLE
Some more Markup edits

### PR DESCRIPTION
**Pull request by tunetheweb**
@AlexLakatos was reading your chapter today and not sure about this section:

> Out of the 6 million desktop pages scanned, an overwhelming 84.4% were compressed with either gzip (62.7%) or Brotli (21.7%) compression. For mobile pages, the numbers are very similar, 85.6% were compressed with either gzip (63.7%) or Brotli (21.9%) compression.
>
> While the slight variation in percentages for mobile and desktop is not surprising, what is surprising is that almost one percentage point more pages are compressed for mobile only. In a mobile world, where every byte of data has a cost associated with it, seeing that mobile pages are not only optimized, but smaller than the desktop counterparts is great. You can learn more about the states of content encoding and the mobile web in the Compression and Mobile Web chapters.

I don't think they are compressed "for mobile mobile only", and think it more likely reflects the fact that different URLs are use for each, and that the mobile data set has a lot more URLs (perhaps more of the long tail that uses "standard" platforms like WordPress that include compression by default?). If those same URLs were visited on desktop they would presumably be compressed too, but they aren't visited enough on desktop to pass our threshold for inclusion.

I also think that a 1% difference is easily explainable with this different dataset differences rather than necessarily a reflection of different usage for mobile and desktop.

I've suggested changing it to this:

> Out of the 6 million desktop pages scanned, an overwhelming 84.4% were compressed with either gzip (62.7%) or Brotli (21.7%) compression. For mobile pages, the numbers are very similar, 85.6% were compressed with either gzip (63.7%) or Brotli (21.9%) compression. The slight variation in percentages for mobile and desktop is not surprising, as they comprise of different URLs, and the [Mobile data set is a lot larger](./methodology#websites).
>
> Compression is important as, particularly in a mobile world, every byte of data has a cost associated with it. You can learn more about the states of content encoding and the mobile web in the [Compression](./compression) and [Mobile Web](./mobile-web) chapters.

I've also removed it from the summary, and make a few other tweaks.

Let me know what you think?

@ksakae1216 I've also made the same edits to the Japanese chapter where I could but need your help on some of them, assuming @AlexLakatos agrees.

### CODE REVIEW

File: src/content/ja/2021/markup.md [hunk 1]
```diff
@@ -115,9 +115,9 @@ _マークアップの章では、HTMLに焦点を当てます。他のマーク
   sql_file="content_encoding.sql"
 ) }}
 
-デスクトップ600万ページのうち、84.4%がgzip（62.7%）またはBrotli（21.7%）で圧縮されていることがわかりました。モバイルページについても、85.6%がgzip（63.7%）またはBrotli（21.9%）で圧縮されており、非常に似たような数字となっています。
+デスクトップ600万ページのうち、84.4%がgzip（62.7%）またはBrotli（21.7%）で圧縮されていることがわかりました。モバイルページについても、85.6%がgzip（63.7%）またはBrotli（21.9%）で圧縮されており、非常に似たような数字となっています。 {# TODO - make same edits here #}
```
Comment (tunetheweb) [inline src/content/ja/2021/markup.md:118]: @ksakae1216 could you suggest the translation here? Here's the suggested new English text:

```md
Out of the 6 million desktop pages scanned, an overwhelming 84.4% were compressed with either gzip (62.7%) or Brotli (21.7%) compression. For mobile pages, the numbers are very similar, 85.6% were compressed with either gzip (63.7%) or Brotli (21.9%) compression. The slight variation in percentages for mobile and desktop is not surprising, as they comprise of different URLs, and the [Mobile data set is a lot larger](./methodology#websites).
```

File: src/content/ja/2021/markup.md [hunk 2]
```diff
@@ -115,9 +115,9 @@ _マークアップの章では、HTMLに焦点を当てます。他のマーク
...
 
-モバイルとデスクトップでパーセンテージが若干異なるのは、驚くことではありませんが、驚くべきは、モバイルのみで圧縮されるページがほぼ1ポイント多いということです。 [データの1バイトごとにコストがかかるモバイルの世界](./mobile-web)では、モバイルのページが最適化されているだけでなく、デスクトップよりも小さいことを確認するのは素晴らしいことです。コンテンツエンコーディングの状態やモバイルウェブについては、[圧縮](./compression) や [モバイルウェブ](./mobile-web) の章で詳しく説明しています。
+モバイルとデスクトップでパーセンテージが若干異なるのは、驚くことではありませんが、驚くべきは、モバイルのみで圧縮されるページがほぼ1ポイント多いということです。 [データの1バイトごとにコストがかかるモバイルの世界](./mobile-web)では、モバイルのページが最適化されているだけでなく、デスクトップよりも小さいことを確認するのは素晴らしいことです。コンテンツエンコーディングの状態やモバイルウェブについては、[圧縮](./compression) や [モバイルウェブ](./mobile-web) の章で詳しく説明しています。 {# TODO - make same edits here #}
```
Comment (tunetheweb) [inline src/content/ja/2021/markup.md:120]: @ksakae1216 here's the suggested new English text:

```md
Compression is important as, particularly in a mobile world, every byte of data has a cost associated with it. You can learn more about the states of content encoding and the mobile web in the [Compression](./compression) and [Mobile Web](./mobile-web) chapters.
```

File: src/content/ja/2021/markup.md [hunk 3]
```diff
@@ -679,7 +679,7 @@ Googleは2021年になってもトップページに`center`要素を使用し
 
 #### メタフレーバー
 
-今年、`meta`要素の人気が落ちてきているので、もう少し詳しく見ていきたいと思います。meta要素は、機械的に読み取り可能な情報をページに追加する方法を提供し、また、いくつかの巧妙なHTTP同等機能を実行します。たとえば、ページのコンテンツセキュリティポリシーを設定できます。
+今年、`meta`要素の人気が落ちてきているので、もう少し詳しく見ていきたいと思います。meta要素は、機械的に読み取り可能な情報をページに追加する方法を提供し、また、いくつかの巧妙なHTTP同等機能を実行します。たとえば、ページのコンテンツセキュリティポリシーを設定できます。 {# TODO make same edits here #}
```
Comment (tunetheweb) [inline src/content/ja/2021/markup.md:682]: @ksakae1216 here's the suggested new English text:

```md
`meta` elements are gaining some of their lost popularity this year, so we wanted to take a closer look at them. They provide a way to add machine-readable information to your pages, as well as perform some nifty HTTP equivalents. For example, setting a _Content Security Policy for a page:
```

**Review by tunetheweb — COMMENTED** [review]
@ksakae1216 if you could suggest the correct Japanese translations of the bits I couldn't edit:

File: src/content/en/2021/markup.md [hunk 4]
```diff
@@ -679,7 +679,7 @@ The second most popular attribute is `href`, with 9.9% of occurrences. With link
 
 #### Meta flavors
 
-`meta` elements are gaining some of their lost popularity this year, so we wanted to take a closer look at them. They provide a way to add machine-readable information to your pages, as well as perform some nifty HTTP equivalents. For example, setting a content security policy for a page:
+`meta` elements are gaining some of their lost popularity this year, so we wanted to take a closer look at them. They provide a way to add machine-readable information to your pages, as well as perform some nifty HTTP equivalents. For example, setting a _Content Security Policy for a page:
```
Comment (AlexLakatos) [inline src/content/en/2021/markup.md:682]:
```suggestion
`meta` elements are gaining some of their lost popularity this year, so we wanted to take a closer look at them. They provide a way to add machine-readable information to your pages, as well as perform some nifty HTTP equivalents. For example, setting a _Content Security Policy_ for a page:
```

**Review by AlexLakatos — APPROVED** [review]
Looks good to me, thanks for catching this @tunetheweb . I've made a minor suggestion, I think there was either an extra _ or a missing one for CSP.

File: src/content/en/2021/markup.md (hunk 4 above)
Comment (tunetheweb) [reply src/content/en/2021/markup.md:682]: Good spot!

File: src/content/ja/2021/markup.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2021/markup.md:118]: @tunetheweb 
`デスクトップ600万ページのうち、84.4%がgzip（62.7%）またはBrotli（21.7%）で圧縮されていることがわかりました。モバイルページについても、85.6%がgzip（63.7%）またはBrotli（21.9%）で圧縮されており、非常によく似た数字となっています。モバイルとデスクトップで割合が若干異なるのは、異なるURLで構成されていることと、[モバイルデータセットがより大きい](./methodology#websites)ことから、驚くべきことではありません。`

File: src/content/ja/2021/markup.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2021/markup.md:120]: @tunetheweb 
`特にモバイルの世界では、1バイトのデータにもコストがかかるため、圧縮は重要です。[圧縮](./compression)と[モバイルウェブ](./mobile-web)の章で、コンテンツエンコーディングとモバイルウェブの状態についてより詳しく知ることができます。`

File: src/content/ja/2021/markup.md (hunk 3 above)
Comment (ksakae1216) [reply src/content/ja/2021/markup.md:682]: @tunetheweb 
`今年、`meta`要素の人気が落ちてきているので、もう少し詳しく見ていきたいと思います。これらは、機械的に読み取れる情報をページに追加する方法を提供するだけでなく、いくつかの巧妙なHTTP同等機能を実行します。たとえば、ページに対して _コンテンツセキュリティポリシー_ を設定できます。`

**Review by ksakae1216 — APPROVED** [review]
I translated from new Englisth text to Japanese.

**Comment by tunetheweb** [comment]
Thanks both! Will merge and release this now.

**Pull request merged**